src/main/java/com/continuum/nova/NovaRenderer.java

```diff
@@ -68,7 +70,7 @@ public NovaRenderer() {
     public void onResourceManagerReload(@Nonnull IResourceManager resourceManager) {
         this.resourceManager = resourceManager;
 
-        if(firstLoad) {
+        if (firstLoad) {
```

Comment:
why do you dislike these spaces so much? I can't think of anyone or any project that uses this particular code style...